package.json

```diff
@@ -35,8 +35,8 @@
     }
   },
   "lint-staged": {
-    "src/**/*.{vue,css}": "stylelint --max-warnings=0",
-    "src/**/*.{vue,js,md}": "eslint --max-warnings=0"
```

Comment:
I think I intentionally changed these to `"src/**/*"` a while back because I felt like lints for Lab/Styleguide/webpack code was distracting, especially since most stuff going into Lab is quick & dirty prototypes. Do you mind leaving these as-is, or do you strongly feel that we need to linting even random hack-ish Lab prototypes?

Reply:
As in, only lint `src` via commit hook and lint lab/styleguide/build via CI? (That's the current behavior except CI linting is broken.)

BTW commit hooks should serve more as a DX improvement than a hindrance. Using no-verify (`-n`) should be acceptable when you expect errors. I do it all the time when I expect linting errors but still prefer to commit.

Reply:
well, I'm saying only lint `src` period. Don't lint lab/styleguide/build at all.

Reply:
Hmm, considering that the purpose of having a consistent style is to improve maintainability for all collaborators, I think all parts of the codebase should be held to the same standard. It's a slippery slope when multiple people start working on the same unlinted file.

This also contributes to one of your goals, which is to set Maker up so that anyone can come in and understand and make changes.

Reply:
Fair enough.